pkg/plugins/policies/meshhealthcheck/plugin/v1alpha1/plugin.go

```diff
@@ -43,13 +43,18 @@ func (p plugin) Apply(rs *core_xds.ResourceSet, _ xds_context.Context, proxy *co
 }
 
 func applyToOutbounds(rules core_xds.ToRules, outboundClusters map[string]*envoy_cluster.Cluster, dataplane *core_mesh.DataplaneResource, routing core_xds.Routing) error {
+	targetedClusters := map[*envoy_cluster.Cluster]string{}
 	for _, outbound := range dataplane.Spec.Networking.GetOutbound() {
 		serviceName := outbound.GetTagsIncludingLegacy()[mesh_proto.ServiceTag]
 		cluster, ok := outboundClusters[serviceName]
 		if !ok {
 			continue
 		}
 
+		targetedClusters[cluster] = serviceName
```

Comment:
same thing happends in https://github.com/kumahq/kuma/blob/6d42311dfad6d30002ac7a3ffb54a8f1113173f6/pkg/plugins/policies/meshcircuitbreaker/plugin/v1alpha1/plugin.go#L95 can this be an issue there as well? Are there other policies that can have the same bug?

Reply:
yeah probably

Reply:
Changed it so this PR doesn't close the issue